src/main/java/com/github/princesslana/eriscasper/rest/guild/BeginGuildPruneRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.github.princesslana.eriscasper.rest.guild;
+
+import com.github.princesslana.eriscasper.util.QueryStringBuilder;
+import org.immutables.value.Value;
+
+/**
+ * @see <a
+ *     href="https://discordapp.com/developers/docs/resources/guild#begin-guild-prune-query-string-params">
+ *     https://discordapp.com/developers/docs/resources/guild#begin-guild-prune-query-string-params</a>
+ */
+@Value.Immutable
+public interface BeginGuildPruneRequest {
+  Integer getDays();
+
+  default String toQueryString() {
+    return new QueryStringBuilder().add("days", getDays().toString()).build();
+  }
+
+  static BeginGuildPruneRequest from(int days) {
```

Comment:
I believe `of` would be a more consistent name used by immutables.

Do we have these methods in these classes elsewhere?

Reply:
in a few of them which only require one param or simple calls, similar to what was done with CreateMessage.

Reply:
Cool. I thought we had put those methods elsewhere. `of` is the more consistent name.

Reply:
Gotcha, I'll change it.